### DISCUSSION TITLE
v1.34.0 Let's Serve

### DISCUSSION THREAD

**Discussion by AndreWohnsland** [AndreWohnsland/CocktailBerry#149] [Announcements]
This update reworks the selection view to cause less confusion for the user. Different alcohol concentration can now be adjusted by two buttons, the serving sizes can be selected directly over buttons, the values (and number of values) can be defined in the config. The spend single ingredient button can now also be displayed as a card in the maker view.

DEPRECATION WARNING: With the latest Raspberry Pi OS, Python 3.11 is shipped as default. In the future, this will also be the required Python version.
Starting early 2024, with Python versions older than 3.11, there will no more updates be available. CocktailBerry will still run as usual.
Please consider upgrading to the latest Raspberry Pi OS and use the backup function to move your data to the latest OS.

# 🆕 New Features of this Release
- New buttons for alcohol level adjustment
- New buttons for multiple serving sizes directly instead of manual adjustment
- New config value to configure spend buttons (MAKER_PREPARE_VOLUME)
- Option to also show spend single ingredient at maker view (MAKER_ADD_SINGLE_INGREDIENT)

# 💎 Improvements
- Proper logging of exact spending of single ingredient if aborted early
- Config window is now separated into tabs containing similar categories instead all in one view

<hr /><em>This discussion was created from the release <a href='https://github.com/AndreWohnsland/CocktailBerry/releases/tag/v1.34.0'>v1.34.0 Let's Serve</a>.</em>

**Comment by Marvin0795** [comment]
Ein Entwickler der hinter seiner Software steht 👍
Vielen Dank für das sinnvolle Update!
